.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Please don't commit the .vscode directory. delete it, commit the delete, and add this directory to the .gitignore

Reply:
fixed in latest commit